test/functional/buffering_proxy.test.js

```diff
@@ -31,7 +32,7 @@ describe.skip('Buffering Proxy', function () {
 
     test: function (done) {
       var configuration = this.configuration;
-      var currentIsMasterIndex = 0;
+      var currentPrimaryIndex = 0;
```

Comment:
It's not clear to me what this variable represents, so I'm not sure if this should be currentPrimaryIndex or helloIndex.